proposals-CG-WG.md

```diff
@@ -1,82 +1,78 @@
-# FedID CG/WG Process Mechanics
+# CG/WG Proposal Stages
+
+Author: @samuelgoto
+Created: 08/11/2024
+Last update: 09/11/2024
+Status: under review
 
 This is a proposal to break proposals into 5 stages of maturity, with clear guidelines and requirements to advance them:
 
-* [Stage 0](#stage0): Understand the Problem Space
-* [Stage 1](#stage1): Incubate Proposals
-* [Stage 2](#stage2): Specify a Preferred Proposal
-* [Stage 3](#stage3): Increase Deployment Confidence
-* [Stage 4](#stage4): Publish a Proposed Recommendation
+* [Stage 0](#stage0): **Exploration** of the Problem Space
+* [Stage 1](#stage1): **Incubation** of competing Alternatives
+* [Stage 2](#stage2): **Formalization** a preferred Proposal
+* [Stage 3](#stage3): **Independent Implementations** of the preferred Proposal
+* [Stage 4](#stage4): **Publication** of a Proposed Recommendation
 
-# Stage 0
+# Stage 0: Exploration
 
-The purpose of Stage 0 proposals is to allow anyone to raise and explore the Problem Space without asking for permission.  
+The purpose of Stage 0 proposals is to allow anyone to raise and explore the Problem Space without asking for permission. 
 
-  * Entrance criteria
-    * Nothing: no permission needed from anyone. 
-  * Entrance artifact
-    * An issue, a personal repo, a blog post, a tweet, etc 
-  * Exit criteria
-    * Identification of [champions](https://github.com/tc39/how-we-work/blob/main/champion.md)
-    * A problem statement 
-# Stage 1
+  * What's needed from individuals?
+    * Typically, a written issue, a personal repo, a blog post, a tweet, etc. 
+  * What's asked of the **Community Group**?
+    * Nothing: no permission needed from anyone.
+
+# Stage 1: Incubation
 
 The purpose of Stage 1 proposals is to explore the Solution Space and pick a preferred direction. This is where most of the work gets done because it involves exploring alternatives, understanding tradeoffs, gathering implementation experience, incubating alternatives, gathering evidence of demand and fitness for purpose and finally identifying the best out of the many alternatives.
```

Comment:
```suggestion
The purpose of Stage 1 proposals is to explore the Solution Space and pick a preferred direction. This is where most of the work gets done because it involves exploring alternatives, understanding tradeoffs, gathering implementation experience, incubating alternatives, gathering evidence of demand and fitness for purpose, and finally, identifying the best out of the many alternatives.
```